Src/DependencyCollector/Shared/Implementation/DependencyTableStore.cs

```diff
@@ -6,13 +6,13 @@
 
     internal class DependencyTableStore : IDisposable
     {
+        internal static bool IsDesktopHttpDiagnosticSourceActivated = false;
```

Comment:
previously it was instance field and request it's value caused expensive creation of DependencyTableStore. It's not needed for diagnostic listener